pkg/postage/batchservice/batchservice.go

```diff
@@ -166,7 +166,7 @@ func (svc *batchService) UpdateDepth(id []byte, depth uint8, normalisedBalance *
 	if err != nil {
 		return fmt.Errorf("get: %w", err)
 	}
-	err = svc.storer.Put(b, normalisedBalance, depth)
+	err = svc.storer.Update(b, normalisedBalance, depth)
```

Comment:
Consider UpdateDepth function. These new functions will greatly reduce complexity and any unnecessary decision making in the batchstore package.

Reply:
Also normalisedBalance is not needed as an arg

Reply:
Don't want to create too many methods. As a first iteration, this should be enough and we can review it later after the changes to the reserve computation.